Makefile

```diff
@@ -0,0 +1,21 @@
+SNAPPY = $(CHAIN)/vendor/github.com/google/snappy
+ROCKSDB = $(CHAIN)/vendor/github.com/facebook/rocksdb
+
+null:
+	@echo Please specify a make target.
+
+## builds chain core with c dependencies, for development environments
+build-dev: build-lib
+	cd $(CHAIN) && go install -tags 'localhost_auth http_ok reset' ./cmd/cored
+
+## builds statically-linked c dependencies
+build-lib:
+	-mkdir $(SNAPPY)/build
```

Comment:
Looks like `$(SNAPPY)/build` is part of the github repo. Does the build process require that we use it, or could we choose a more-obviously-temporary name for the build directory? It would be nice not to create git diffs with every run of `make build-lib` or `make clean`.

Maybe the right thing is to remove that `build` subdir from the repo? (Looks like it, because its `CMakeCache.txt` includes directory names under `/Users/tessr`.)